dask/dataframe/rolling.py

```diff
@@ -292,8 +292,19 @@ def quantile(self, quantile):
         return self._call_method('quantile', quantile)
 
     @derived_from(pd_Rolling)
-    def apply(self, func, args=(), kwargs={}):
-        return self._call_method('apply', func, args=args, kwargs=kwargs)
+    def apply(self, func, args=(), kwargs={}, **kwds):
+        # TODO: In a future version of pandas this will change to
+        # raw=False. Think about inspecting the function signature and setting
+        # to that?
+        if PANDAS_VERSION >= '0.23.0':
```

Comment:
This isn't 100% future-proof. I'm not sure it's worth trying to inspect the function, especially since pandas doesn't publicly expose `pandas.core.window.Rolling` / `Expanding` objects.